### ISSUE TITLE
Where is the Indian_best_3DCONV.hdf5

### ISSUE THREAD

**Issue by IceStreams** [zilongzhong/SSRN#19]
Hi,where is the file Indian_best_3DCONV.hdf5
the code you have writen in the Real3D_Conv.py
best_weights_path = '/home/finoa/DL-on-HSI-Classification/Best_models/Indian_best_3DCONV.hdf5'

**Comment by zilongzhong** [comment]
Here it is:
https://github.com/zilongzhong/SSRN/blob/master/models/Indian_best_RES_3D_SS4_10_1.hdf5

Best reg

**Comment by IceStreams** [comment]
Can I create a new  empty file named Indian_best_3DCONV.hdf5 in the file of models and then Run the program  Real3D_Conv.py?

**Comment by zilongzhong** [comment]
No, you don't need to. This is the best performed trained model that is saved during training, just make sure the model is constructed correctly.
